bundles/org.eclipse.equinox.p2.director.app/src/org/eclipse/equinox/internal/p2/director/app/messages.properties

```diff
@@ -88,4 +88,5 @@ File_does_not_exist=File does not exist: {0}.
 Cannot_set_iu_profile_property_iu_does_not_exist=Unable to set IU profile properties because the following IU does not exist: {0}.
 Unmatched_iu_profile_property_key_value=Unmatched IU profile property key/value pair: {0}.
 Bad_format=Bad format ({0}) in IU profile properties file: {1}.
-Cant_write_in_destination=The operation you've requested can not be performed because the folder {0} is read only.
\ No newline at end of file
+Cant_write_in_destination=The operation you've requested can not be performed because the folder {0} is read only.
+Help_Add_Jre_IU=Include Default JRE Installable Unit as an extra installable Unit. This IU acts as dependency for actual IU's being installed does not install/provide JRE artifact.
```

Comment:
The grammar could be improved.  Maybe something like this:

Include a default JRE installable unit as an extra IU. This IU satisfies JRE dependencies in the case that the content metadata does not otherwise provide an IU for that purpose.